### ISSUE TITLE
Tool and dataset information disappears when moving pointer outside of the (more) hypertext links

### ISSUE THREAD

**Issue by bryancaron** [aces/cbrain#1433]
When viewing the tool and dataset listing page at https://portal.cbrain.mcgill.ca/available/ additional details for each tool or dataset are viewable but only when the pointer is directly over the (more) hypertext link.  As soon as the pointer is moved outside of the range of screen covered by (more) then the information disappears.  This in turn makes it difficult for the user to see the information as a persistent source when they are looking to move back and forth between browser windows, for example.

It would be helpful if the information box displayed would be persistent such that it remains visible even when the the pointer position is changed, either within the same browser window as the listing, or to another browser window, until it is explicitly closed by the user.

![Screenshot 2024-09-19 at 2 29 31 PM](https://github.com/user-attachments/assets/8e6222fe-872a-467f-bd94-6b008612fa3e)

The behaviour as seen when clicking on the "Help" button when Launching a task is more akin to a desired behaviour in that it persists until the user closes that particular information box.

Bryan.

**Comment by prioux** [comment]
Yes, these overlay panels are very old and annoying to no end. I created them. They were supposed to be used only for short quick tiny bits of informations (e.g. when hovering over a username, get their city and last login).

For the available page, we need a better more persistent box. I think we have something like that in CBRAIN where a separate panel opens, but I don't know the name of the helper that does that.

**Comment by MontrealSergiy** [comment]
- on task edit page, parameter help opens a full-window large 'modal' box, is it this panel? helper seems to be called `overlay` smth
- alternatively  we can just expand cell on click, like it is done in message list
- pretty often click more cell or page elements is shown (e.g. when browsing nested folders) in file  collection show page

**Comment by prioux** [comment]
Closed by #1449
